### ISSUE TITLE
[Export CSV] Remove "data:" prefix from submission columns?

### ISSUE THREAD

**Issue by gino-m** [google/ground-platform#1979]

**Comment by gino-m** [comment]
@lecrabe @jo-spek Do you think these prefixes are more helpful or harmful?

**Comment by gino-m** [comment]
@lecrabe @jo-spek @kenstershiro Gentle ping

**Comment by jo-spek** [comment]
I think they could come in handy for certain sorting cases. What I find confusing though, is that "geometry" and "name" have no prefix and that "draw or walk a perimeter" is kept as an empty data field. I'd argue to go for what's easiest. If the easiest is to just drop the prefixes, I'm all for it, but let's wait for another opinion.

**Comment by gino-m** [comment]
> I think they could come in handy for certain sorting cases. What I find confusing though, is that "geometry" and "name" have no prefix and that "draw or walk a perimeter" is kept as an empty data field. I'd argue to go for what's easiest. If the easiest is to just drop the prefixes, I'm all for it, but let's wait for another opinion.

What do you mean "draw or walk a perimeter" is kept as an empty data field?

I think the choice of column names depends on how data will be analyzed, and if there are any limitations. Another option is to let the user exporting the data to edit the column names (#769), in which case the question about prefix would be moot.

**Comment by jo-spek** [comment]
> 
> What do you mean "draw or walk a perimeter" is kept as an empty data field?
> 
> I think the choice of column names depends on how data will be analyzed, and if there are any limitations. Another option is to let the user exporting the data to edit the column names (#769), in which case the question about prefix would be moot.

Looking at the screenshot you first posted, there is the geometry column, but then also a column "data:Draw or walk perimeter" which is empty. I find that a bit confusing. 

However, overall I think that those handling the exported CSVs will be a bit more tech-savvy than the data collectors and will be able to adjust the column titles to their needs in the exported CSV. I think #769 would be an extra step that nobody desperately needs. I'd vouch to just keep it as it is.

**Comment by gino-m** [comment]
> Looking at the screenshot you first posted, there is the geometry column, but then also a column "data:Draw or walk perimeter" which is empty. I find that a bit confusing.

I see, I think that was a bit of a red herring - that was an example of "map annotations", where data collectors can mark up a site with points and polygons. That feature is currently disabled in the app, so please disregard that column.

> However, overall I think that those handling the exported CSVs will be a bit more tech-savvy than the data collectors and will be able to adjust the column titles to their needs in the exported CSV. I think #769 would be an extra step that nobody desperately needs. I'd vouch to just keep it as it is.

Sgtm..moving that issue to Icebox!

**Comment by kenstershiro** [comment]
Agreed that allowing users to specify column names (#769) is probably overkill, if they want to they can always edit the exported csv. As for the data: prefix, as a first time user I found it a bit confusing and redundant - it's practically on all the columns apart from the first couple on the site location, so I don't really see the need for it. @jo-spek you mention it could be useful for sorting - do you prefer to leave as is? I'll move this to icebox for now but we can delete this issue if you think the prefix is useful.

**Comment by jo-spek** [comment]
My thought was that it could be useful if there were different categories and it seemed to me as if this was also an interoperability thing, making fields more easily readable for other softwares. If the latter is not the case, I'd vouch for deprecating it entirely. That said, I think that even deprecating it is very low priority at the moment, so let's keep it in the icebox.

**Comment by gino-m** [comment]
Not that I know of. I added it to differentiate fields which pertain to the LOI (aka data collection site). Those fields can be repeated in multiple rows, since the export is equivalent to a left outer join of LOIs and submissions. If you think it's worth addressing in H1 2025 I'd keep it in GA, otherwise move to Icebox makes sense.
